src/sentry/search/utils.py

```diff
@@ -236,9 +236,10 @@ def tokenize_query(query):
             continue
 
         key, value = token.split(':', 1)
+        real_key = key[1:] if key and key[0] == '-' else key
```

Comment:
This can be moved into the conditional below since it's only used within that block.